### DISCUSSION TITLE
what can be the case?

### DISCUSSION THREAD

**Discussion by sinaainesaz** [PMEAL/OpenPNM#2873] [Q&A]
here are the relperm curves of my 205 networks. the blue is the wetting phase which is water. i did my best to add as much diversity as i can to the curves. i have been successfull for the other phase which is air and I call it oil and i have assigned the parameters like viscosity, surface tension of oil to it. but i was not successful to have diverse change for my water relative permeability curves. I know I have been asking lot's of questions. but please tell me why i have this much trouble with the water relperm cruves.
![image](https://github.com/PMEAL/OpenPNM/assets/81174257/310a3c38-83f2-4842-9737-9fe4dcb70658)

**Comment by jgostick** [comment]
So you are saying that when using the `Water` class, things don't work?  What happens if you use a generic `Phase` class and add the properties of water?
